pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -457,6 +469,8 @@ xdvi = stdenv.mkDerivation {
   # TODO: it's suspicious that mktexpk generates fonts into ~/.texlive2014
 };
 
+context = callPackage ./lmtx.nix {};
```

Comment:
In line with other packages, I supposed we should inline `lmtx.nix`, and avoid `callPackage`.